pages/search.tsx

```diff
@@ -114,6 +108,14 @@ const Search: NextPage = () => {
     })
   }
 
+  const onPaginationChange = (event: MouseEvent | KeyboardEvent | null, page: number) => {
+    const offset = page - 1
+    router.replace(
+      { query: { ...router.query, offset } },
+      { query: { ...router.query, offset } },
```

Comment:
what's going on here? 🤔

Reply:
I want offset to be visible in the url? And would like shallow